Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM educatedwarrior/invictus_image:1.59
+MAINTAINER educatedwarrior 
+
+# Configuration variables
+#test or prod for NODE_TYPE
+ENV NODE_TYPE prod
+ENV LANG en_US.UTF-8
+ENV WORKDIR /opt/muse/bin
+ENV DATADIR /opt/muse/bin/witness_node_data_dir
+ENV USEGENSISJSON false 
+ENV TEST_SEED 192.34.60.157:29092
+ENV PROD_SEED 138.197.68.175:33333
+
+LABEL org.freenas.interactive="false"       \
+      org.freenas.version="1.59.1.0002"      \
+      org.freenas.upgradeable="true"        \
+      org.freenas.expose-ports-at-host="true"   \
+      org.freenas.autostart="true"      \
+      org.freenas.web-ui-protocol="http"    \
+      org.freenas.web-ui-port=8090     \
+      org.freenas.web-ui-path=""     \
+      org.freenas.port-mappings="8090:8090/tcp"         \
+      org.freenas.volumes="[                    \
+          {                         \
+              \"name\": \"${DATADIR}\",              \
+              \"descr\": \"Data directory\"           \
+          }                         \
+      ]"                            \
+      org.freenas.settings="[                   \
+          {                         \
+              \"env\": \"NODE_TYPE\",                  \
+              \"descr\": \"test or prod.  Default value test\",       \
+              \"optional\": false                \
+          },                            \
+          {                         \
+              \"env\": \"TEST_SEED\",                \
+              \"descr\": \"Default value 192.34.60.157:29092\",          \
+              \"optional\": true                \
+          },                            \
+          {                         \
+              \"env\": \"PROD_SEED\",                \
+              \"descr\": \"Default value 138.197.68.175:29092\",         \
+              \"optional\": true                \
+          }                            \
+      ]"
+
+#Build blockchain source
+RUN \
+	cd /tmp && git clone https://github.com/themuseblockchain/Muse-Source.git && \
+	cd Muse-Source && \
+	git submodule update --init --recursive && \
+	cmake -j 8 -DBOOST_ROOT="$BOOST_ROOT" -DBUILD_MUSE_TEST=OFF -DCMAKE_BUILD_TYPE=Debug . && \
```

Comment:
Use -DCMAKE_BUILD_TYPE=Release instead?

Reply:
Can be both, if you wanna attach gdb/lldb for any reason debug is required.